src/LeanCode.ContractsGenerator.Tests/LeanCode.ContractsGenerator.Tests.csproj

```diff
@@ -6,28 +6,32 @@
     <IsPackable>false</IsPackable>
   </PropertyGroup>
 
-  <Target Name="PackDependencies" BeforeTargets="Build" Condition="'$(IsInnerBuild)' != 'true'">
-    <MSBuild Projects="../LeanCode.Contracts/LeanCode.Contracts.csproj" Properties="Configuration=Debug" Targets="Restore" />
-    <MSBuild Projects="../LeanCode.Contracts/LeanCode.Contracts.csproj" Targets="Build;Pack" />
+  <ItemGroup>
+    <FxToCompile Include="$(TargetFrameworks)" />
+  </ItemGroup>
+
+  <Target Name="PackDependenciesForTests" BeforeTargets="AfterBuild">
+    <Message Text="Packing the LeanCode.Contracts" Importance="high" />
+    <MSBuild
+      StopOnFirstFailure="true"
+      Projects="../LeanCode.Contracts/LeanCode.Contracts.csproj"
+      Properties="Configuration=Debug;TargetFramework=net6.0;TargetFrameworks=net6.0;Version=9.9.9.9-internal"
```

Comment:
Why not use `$(Version)` here instead of `9.9.9.9-internal`?

Reply:
To mark that this is deployed just for test and not some normal version. Using `Version` confuses NuGet if there is a different (lower/higher) version available elsewhere.